SoftLayer/CLI/hardware/detail.py

```diff
@@ -71,9 +72,11 @@ def cli(env, identifier, passwords, price, components):
     table.add_row(['last_transaction', last_transaction])
     table.add_row(['billing', 'Hourly' if result['hourlyBillingFlag'] else 'Monthly'])
 
-    vlan_table = formatting.Table(['type', 'number', 'id'])
+    vlan_table = formatting.Table(['type', 'number', 'id', 'Name', 'netmask'])
```

Comment:
If the field Name is in capital letter, maybe the rest of the fields should the same format